.gitlab/ci/.gitlab-ci.on-push.yml

```diff
@@ -452,6 +452,12 @@ slack-notify-nightly-build:
     - echo "Authenticating GCP"
     - gcloud auth activate-service-account --key-file="$GCS_ARTIFACTS_KEY" > $ARTIFACTS_FOLDER/logs/gauth.out 2>$ARTIFACTS_FOLDER/logs/gauth.err
     - echo "Auth done successfully"
+    - |
+      if [[ -n "${NIGHTLY}" ]]; then
+        echo "Setting CLOUD_MACHINES_TYPE to nightly"
+        CLOUD_MACHINES_TYPE="nightly"
+        GCS_LOCKS_PATH="content-locks/locks-xsiam-ga-nightly"
+      fi
```

Comment:
The step `.test_content_on_cloud_server_instances_base:` is used for XSIAM and XSOAR-NG. 
Once the XSOAR-NG Nightly is merged, this part of the code will have to be changed as well. (and we may forget it to change it).
 In general, because this step uses us for all cloud instances: XSIAM and XSOAR-NG we cant implement a logic relevant only for XSIAM here.

Please check if we can use [Global Variables with if conditions](https://docs.gitlab.com/ee/ci/yaml/#rulesvariables), and set the `CLOUD_MACHINES_TYPE` and `GCS_LOCKS_PATH` inside `xsiam_server_ga`.

Reply:
If the rules variables will work, please change it here as well: https://github.com/demisto/content/blob/dd4b6519654d1232bcc7b768b0fc3b759155c8e7/.gitlab/ci/.gitlab-ci.bucket-upload.yml#L217